packages/nextra-theme-docs/src/types.ts

```diff
@@ -8,7 +8,7 @@ export interface DocsThemeConfig {
   titleSuffix?:
     | React.ReactNode
     | React.FC<{
-        locale: string
+        locale?: string
```

Comment:
locale now always provided so you need to replace all `locale?: string` by `locale: string`